### PR TITLE
Adding overlay of Esup-Filemanager Portlet 

### PR DESCRIPTION
**Pull request by jgribonvald**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [ ] tests are included
-   [ ] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

from issue #167
The filemanager is integrated into welcome layout and 2 kind of drives are connected, one is a free access ftp, (so readonly) and the second is configured on java.io.tempdir for read/write access demo.
If someone has a better connection point (easy to configure) for a demo with pictures of small videos (to demonstrate embeded media readers) on will be welcome !

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
Hey @jgribonvald!
I ran
```sh
git remote add jgribonvald https://github.com/jgribonvald/uPortal-start
git checkout -t jgribonvald/Add_Esup-Filemanager
./gradlew portalInit portalOpen
```
But am not seeing the portlet on the welcome page
screencapture-localhost-8080-uportal-f-welcome-normal-render-up-2018-05-30-09_00_16

and am not seeing it listed as an add-able portlet from portlet administration.

**Comment by jgribonvald** [comment]
Oh sorry I forgot to add on file, the one that permit to declare the overlay :stuck_out_tongue:

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<portlet-definition version="4.0" xsi:schemaLocation="https://source.jasig.org/schemas/uportal/io/portlet-definition https://source.jasig.org/schemas/uportal/io/portlet-definition/portlet-definition-4.0.xsd" xmlns:ns2="https://source.jasig.org/schemas/uportal" xmlns="https://source.jasig.org/schemas/uportal/io/portlet-definition" xmlns:ns4="https://source.jasig.org/schemas/uportal/io/portlet-type" xmlns:ns3="https://source.jasig.org/schemas/uportal/io/subscribed-fragment" xmlns:ns5="https://source.jasig.org/schemas/uportal/io/user" xmlns:ns6="https://source.jasig.org/schemas/uportal/io/event-aggregation" xmlns:ns7="https://source.jasig.org/schemas/uportal/io/stylesheet-descriptor" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ns8="https://source.jasig.org/schemas/uportal/io/permission-owner">
+    <title>My drives</title>
+    <name>My drives</name>
```
Comment (drewwills) [inline data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: Suggest "My Drives" to match capitalization in other portlet definitions.

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <fname>esup-filemanager-demo</fname>
+    <desc>Drives access for the demonstraction</desc>
```
Comment (drewwills) [inline data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: ### dem·on·strac·tion
/ˌdemənˈstrāSH(ə)n/
noun
1.
The action or process of exhibiting an attractive or working software feature for the purpose of diverting the audience away from what is unattractive or not working in the software.
"it is not capable of mathematical demonstration"

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    <type>Portlet</type>
+    <timeout>3600000</timeout>
```
Comment (drewwills) [inline data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: Really?

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    <portlet-descriptor>
+        <ns2:webAppName>/esup-filemanager</ns2:webAppName>
+        <ns2:portletName>esup-filemanager</ns2:portletName>
+    </portlet-descriptor>
+    <category>Demonstration</category>
+    <category>Work</category>
```
Comment (drewwills) [inline data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: I'm not sure this category currently exists (in the default data set).  Maybe we should simply use `Services`.

**Comment by drewwills** [comment]
I'm happy to see this addition. :+1:

**Comment by ChristianMurphy** [comment]
It's up and running now, thanks @jgribonvald!

my-drives-demo

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml (hunk 1 above)
Comment (jgribonvald) [reply data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: Ok changed in commit override

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml (hunk 4 above)
Comment (jgribonvald) [reply data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: here are all categories:
``` shell
uPortal-start/data$ grep -R "<category>" * | cut -d ":" -f2 | sort -u
    <category>Academics</category>
    <category>Administration</category>
    <category>Demonstration</category>
    <category>Demonstration</category> -->
    <category>Development</category>
    <category>Entertainment</category>
    <category>Featured</category>
    <category>Information</category>
    <category>Instructors</category>
    <category>News</category>
    <category>Services</category>
    <category>Tenant Portlets</category>
    <category>Testing</category>
    <category>uPortal</category>
    <category>Work</category>
```

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml (hunk 2 above)
Comment (jgribonvald) [reply data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: So I purposed something other ! Let me know if you have a better description.

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml (hunk 3 above)
Comment (jgribonvald) [reply data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: 1 hour yes in my production conf.

File: data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml (hunk 2 above)
Comment (ChristianMurphy) [reply data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: > Drives access for the demonstration

is the phrase you are looking for :slightly_smiling_face:

Comment (drewwills) [reply data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: @jgribonvald -- It's a joke... please pardon me for this moment of fun.  :)

AFAIK there is no such word as _demonstraction_;  I made up the dictionary-style definition.

Comment (jgribonvald) [reply data/quickstart/portlet-definition/esup-filemanager-demo.portlet-definition.xml]: Drew I understood for the joke :blush: 
but after removing the bad character, more seriously I've found a better description :thinking: maybe ?

**Comment by ChristianMurphy** [comment]
Thanks @jgribonvald! :bow:

**Pull request merged**